### PR TITLE
Publish three new analyzers

### PR DESCRIPTION
**Pull request by ejball**
Adds FL0024, FL0025, and FL0026.

### CODE REVIEW

File: tests/Faithlife.Analyzers.Tests/ExpressionBodiedMethodArrowTests.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	public void ValidExpressionBodiedProperty()
+	{
+		const string validProgram = """
+			class Test
+			{
+				public int Value
+					=> 1;
+			}
+			""";
+		VerifyCSharpDiagnostic(validProgram);
```
Comment (bgrainger) [inline tests/Faithlife.Analyzers.Tests/ExpressionBodiedMethodArrowTests.cs]: Why no diagnostic?

Comment (ejball) [reply tests/Faithlife.Analyzers.Tests/ExpressionBodiedMethodArrowTests.cs]: I shouldn't assign reviewers before I review the code myself, should I... 😬 

It took my instructions literally, which said "methods". I'll fix it; thanks for catching it.

Comment (bgrainger) [reply tests/Faithlife.Analyzers.Tests/ExpressionBodiedMethodArrowTests.cs]: If expression-bodied properties are out of scope for this PR then this test isn't necessary; delete it. It's not verifying correct or incorrect behavior.

But why not enforce style for them at the same time? Seems like the implementation should be extended to also cover properties.

**Review by bgrainger — CHANGES_REQUESTED** [review]
Either fix properties (recommended) or remove the irrelevant test.

File: tests/Faithlife.Analyzers.Tests/ExpressionBodiedMemberArrowTests.cs [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+	[Test]
+	public void ValidLambdaExpression()
+	{
+		const string validProgram = """
+			using System;
+
+			class Test
+			{
+				private readonly Func<int, int> increment = x
+					=> x + 1;
```
Comment (bgrainger) [inline tests/Faithlife.Analyzers.Tests/ExpressionBodiedMemberArrowTests.cs]: I could be convinced that this case should be handled, too. But we could merge without it.

It's nice to have this "documentation" that this analyzer doesn't currently look for this pattern, as long as no one takes it as being prescriptive that it never _should_.

**Review by bgrainger — APPROVED** [review]
The tests demonstrate the detections and fixes I would like to see.

File: tests/Faithlife.Analyzers.Tests/ExpressionBodiedMemberArrowTests.cs (hunk 2 above)
Comment (ejball) [reply tests/Faithlife.Analyzers.Tests/ExpressionBodiedMemberArrowTests.cs]: @bgrainger If you agree, I'll update the analyzer to refer to the "lambda operator" instead of "expression-bodied member arrow" and apply the rule to all three uses:

- lambda expressions
- expression-bodied members
- switch expressions

Comment (bgrainger) [reply tests/Faithlife.Analyzers.Tests/ExpressionBodiedMemberArrowTests.cs]: Sounds good to me to handle all those consistently. 👍

**Comment by ejball** [comment]
@copilot Rename and update the analyzer to refer to the "lambda operator" instead of the "expression-bodied member arrow" and apply the rule to all three uses:

- lambda expressions
- expression-bodied members
- switch expressions

**Review by bgrainger — APPROVED** [review]

File: ReleaseNotes.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 1.7.0
+
+* Add [FL0024](https://github.com/Faithlife/FaithlifeAnalyzers/blob/master/docs/FL0024.md): Lambda operators should end the previous line.
```
Comment (bgrainger) [inline ReleaseNotes.md:5]: Should also include FL0026 in the release notes.

Comment (ejball) [reply ReleaseNotes.md:5]: Done. But maybe I should throw FL0025 in here...

**Comment by ejball** [comment]
Added one more analyzer, since it was already written.

**Review by bgrainger — APPROVED** [review]
PrivateFieldsLastTests look good as far as they go; hopefully we don't have too many edge cases in practice that the tests didn't catch? Easy enough to update it with more examples later.
